### PR TITLE
read-only example

### PR DESCRIPTION
**Pull request by andrew-devs**
Read-only example

### CODE REVIEW

File: howtos/how-to-lock-a-conversation-to-read-only/README.md.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+This is an example project for TalkJS's tutorial on [How to lock a conversation to be read-only](https://talkjs.com/resources/how-to-lock-a-conversation-to-be-read-only/).
+
+The project uses TalkJS's [custom conversation actions](https://talkjs.com/docs/Features/Customizations/Conversation_Actions/) to add a new "Set Read-Only" option to the chat inbox UI. We add code to listen for this custom action and use the [ConversationBuilder.setParticipant](https://talkjs.com/docs/Reference/JavaScript_Chat_SDK/ConversationBuilder/#ConversationBuilder__setParticipant) method to set each participant's access level to read-only.
```
Comment (keerlu) [inline howtos/how-to-lock-a-conversation-to-read-only/README.md.txt:3]: We're planning to standardize on sentence case, so would be good to use it here
```suggestion
The project uses TalkJS's [custom conversation actions](https://talkjs.com/docs/Features/Customizations/Conversation_Actions/) to add a new "Set read-only" option to the chat inbox UI. We add code to listen for this custom action and use the [ConversationBuilder.setParticipant](https://talkjs.com/docs/Reference/JavaScript_Chat_SDK/ConversationBuilder/#ConversationBuilder__setParticipant) method to set each participant's access level to read-only.

```

**Review by keerlu — COMMENTED** [review]
Just run this and tested this works - overall looks good but I'll also try and get a more thorough dev review.

General comment: it might be good to include an extra user to demonstrate locking a group rather than one-to-one conversation. (I've edited this, ignore my previous overcomplicated suggestion!)

File: howtos/how-to-lock-a-conversation-to-read-only/index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    var inbox = talkSession.createInbox({ selected: conversation });
+    inbox.mount(document.getElementById('talkjs-container'));
+
+    inbox.onCustomConversationAction('setReadOnly', (event) => {
+      console.log('Locked conversation with id:', event.conversation.id);
+      inbox.destroy();
+      conversation = talkSession.getOrCreateConversation(event.conversation.id);
+      conversation.setParticipant(me, { access: 'Read' });
+      conversation.setParticipant(other, { access: 'Read' });
+      inbox = talkSession.createInbox({ selected: conversation });
```
Comment (bigblind) [inline howtos/how-to-lock-a-conversation-to-read-only/index.html:75]: I'd call `inbox.select(conversation)` instead of recreating the inbox here.

**Review by bigblind — APPROVED** [review]
Looks good, just one minor change I'd make.

File: howtos/how-to-lock-a-conversation-to-read-only/index.html (hunk 2 above)
Comment (keerlu) [reply howtos/how-to-lock-a-conversation-to-read-only/index.html]: This has now been updated

**Pull request merged**